packaging/build/Microsoft.NET.Build.Containers.targets

```diff
@@ -39,7 +39,7 @@
         <!-- Container Defaults -->
         <PropertyGroup>
             <ContainerBaseImage Condition="'$(ContainerBaseImage)' == ''">$(_ContainerBaseRegistry)/$(_ContainerBaseImageName):$(_ContainerBaseImageTag)</ContainerBaseImage>
-            <ContainerRegistry Condition="'$(ContainerRegistry)' == ''">docker://</ContainerRegistry>
+            <!-- ContainerRegistry is a thing, and it defaults to empty to signal no registry - i.e. a local daemon image. -->
```

Comment:
```suggestion
            <!-- An empty ContainerRegistry implies pushing to the local daemon -->
            <ContainerRegistry Condition="'$(ContainerRegistry)' == ''"></ContainerRegistry>
```
Sort of a style-nit here. ContainerRegistry is an important property, and I feel like defining it (even if empty) is a clearer signal that it exists.

Reply:
I waffled on this - we should take it separately!